### PR TITLE
Story/validate diarrhoea 52

### PR DESCRIPTION
**Pull request by andriikhlivniuk**
Validating diarrhoea by asking questions. User can answer yes or no.

### CODE REVIEW

**Comment by blottn** [comment]
This is missing some files and can't compile yet. Make sure to include the `activity_validate_diarrhoea` layout.

File: app/src/main/java/org/inspire/breath/validateDiarrhoea.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                    answerNo.setVisibility(View.GONE);
+                }
+
+
+            }
+        });
+
+        answerNo.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
```
Comment (WhelanB) [inline app/src/main/java/org/inspire/breath/validateDiarrhoea.java]: Can this be reduced to just the inner 3 lines using the index? The if/else checks are redundant

**Comment by andriikhlivniuk** [comment]
Yes. I have changed this and pushed again. Can you review it, please?

File: app/src/main/java/org/inspire/breath/validateDiarrhoea.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
+package org.inspire.breath;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+import android.widget.Button;
+
+public class validateDiarrhoea extends AppCompatActivity {
```
Comment (WhelanB) [inline app/src/main/java/org/inspire/breath/validateDiarrhoea.java]: could you move this class into org.inspire.breath.activities (see master branch)?

**Review by WhelanB — CHANGES_REQUESTED** [review]

**Comment by andriikhlivniuk** [comment]
Moved class as asked

пн, 11 марта 2019 г. в 20:33, Brian Whelan <notifications@github.com>:

**Comment by blottn** [comment]
Closing Duplicate

**Pull request closed without merging**